src/common/NoResults/styled.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+import { ReactComponent as Picture } from "../../images/Picture.svg"
+
+export const StyledHeader = styled.div`
+    margin: 56px;
```

Comment:
Jeżeli damy tu z każdej strony margines 56px i jeszcze wsadzimy całość w Container, to wydaje mi się, że odstępy mogą się trochę poprzesuwać, ale najwyżej jak coś się popsuje to poprawimy to po złożeniu wszystkich komponentów 😄